### DISCUSSION TITLE
How the RTL backup work

### DISCUSSION THREAD

**Discussion by xbukorg** [Ride-The-Lightning/RTL#1150] [Q&A] [answered]
Hi ,

I have set in RTL-Config.json backup folder to:

"channelBackupPath": "J:\\RTL\\backup\\node-xxxxx",

but created file channel-all.bak have 0KB.

How it work?

Thanx

**Comment by ShahanaFarooqui** [comment, marked as answer]
Channel backup only works for LND node.
